cabal-install/src/Distribution/Client/Config.hs

```diff
@@ -568,6 +572,7 @@ instance Semigroup SavedConfig where
         UploadFlags
           { uploadCandidate = combine uploadCandidate
           , uploadDoc = combine uploadDoc
+          , uploadToken = combine uploadToken
```

Comment:
what happens if both fields are non empty?

Reply:
When multiple tokens are set in the config file it uses the last token. If a token is set in the command line it takes precedence over the token in the config file. If multiple tokens are set in the command line, like in the config file, the last one is used. If both fields are non empty it chooses the last field and discards the first.